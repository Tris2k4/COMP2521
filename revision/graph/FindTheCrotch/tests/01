15
15
1-2 1-3 2-5 3-4 4-5 4-8 8-6 7-6 6-9 6-0 0-11 11-13 11-12 13-14 12-10