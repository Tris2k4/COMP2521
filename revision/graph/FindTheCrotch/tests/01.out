Enter number of vertices: Enter number of edges: Enter edges in the form v-w: 
Graph: nV = 15
Edges:
 0: 0-6 0-11
 1: 1-2 1-3
 2: 2-1 2-5
 3: 3-1 3-4
 4: 4-3 4-5 4-8
 5: 5-2 5-4
 6: 6-0 6-7 6-8 6-9
 7: 7-6
 8: 8-4 8-6
 9: 9-6
10: 10-12
11: 11-0 11-12 11-13
12: 12-10 12-11
13: 13-11 13-14
14: 14-13

The crotch is at vertex 11.
